src/test/java/com/google/devtools/build/lib/remote/GrpcRemoteCacheTest.java

```diff
@@ -1016,4 +1016,59 @@ public void read(ReadRequest request, StreamObserver<ReadResponse> responseObser
       assertThat(st.getCode()).isEqualTo(Status.Code.DEADLINE_EXCEEDED);
     }
   }
+
+  @Test
+  public void isRemoteCacheOptionsWhenGrpcEnabled() {
+    RemoteOptions options = Options.getDefaults(RemoteOptions.class);
+    options.remoteCache = "grpc://some-host.com";
+
+    assertThat(GrpcRemoteCache.isRemoteCacheOptions(options)).isTrue();
```

Comment:
also check that http is not enabled?

Reply:
It is checked in `SimpleBlobStoreFactoryTest`